style.css

```diff
@@ -179,6 +177,154 @@ body {
   display: none;
 }
 
+.second {
+  background-color: #FAE8E4;
+  height: 900px;
+  width: 100%;
+}
+
+.second-container {
+  display: grid;
+  grid-template-columns: 730px 488px;
+  grid-template-rows: 593px; /*622*/
+  grid-column-gap: 42px;
+  padding-top: 172px;
+  padding-left: 106px;
+}
+
+.second-container__text-white {
+  width: 488px;
+  height: 556px;
+  background-color: #fff;
+  box-shadow: 8px 8px 25px rgba(122,122,122,0.5);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)